spring-cloud-modules/spring-cloud-contract/pom.xml

```diff
@@ -60,7 +60,7 @@
     </dependencyManagement>
 
     <properties>
```

Comment:
Upgrade of the spring-cloud-contract-verifier, to bring the newest groovy version needed with JDK 17